package.json

```diff
@@ -2,24 +2,33 @@
   "name": "protobuf-es",
   "private": true,
   "workspaces": [
-    "./packages/protobuf",
-    "./packages/protoc-gen-es",
-    "./packages/protoplugin",
-    "./packages/protoplugin-test",
-    "./packages/protoplugin-example",
-    "./packages/protobuf-test",
-    "./packages/protobuf-conformance",
-    "./packages/bundle-size",
-    "./packages/protobuf-example",
-    "./packages/upstream-protobuf",
-    "./packages/typescript-compat/*"
+    "packages/protobuf",
```

Comment:
Shouldn't be necessary, filed https://github.com/vercel/turbo/issues/8599. It's a harmless change though, and works well enough as a workaround.